### PR TITLE
Add Symfony 3 support

### PR DESCRIPTION
**Pull request by WouterJ**
fix #249

### CODE REVIEW

File: Block/ActionBlockService.php [hunk 1]
```diff
@@ -72,7 +64,7 @@ public function execute(BlockContextInterface $blockContext, Response $response
             return new Response();
         }
 
-        $requestParams = $block->resolveRequestParams($this->request, $blockContext);
+        $requestParams = $block->resolveRequestParams($this->requestStack->getMasterRequest(), $blockContext);
```
Comment (WouterJ) [inline Block/ActionBlockService.php]: need some help here whether we really want the master request or maybe the sub request? /cc @dbu

File: Block/PhpcrBlockLoader.php [hunk 2]
```diff
@@ -220,10 +217,8 @@ protected function determineAbsolutePath($name)
             return $name;
         }
 
-        if ($this->request
-            && $this->request->attributes->has('contentDocument')
-        ) {
-            $currentPage = $this->request->attributes->get('contentDocument');
+        if ($this->requestStack->getMasterRequest()->attributes->has('contentDocument')) {
```
Comment (WouterJ) [inline Block/PhpcrBlockLoader.php]: and the same question here

File: Block/ActionBlockService.php (hunk 1 above)
Comment (dbu) [inline Block/ActionBlockService.php]: the previous code set the request when the service is fetched. but i think it originally used the cmf_request_aware tag for that... i would take the sub request i think. that way, the user has more control over what parameters go around. 

i think that this was currently already the case, unless several action blocks are used and the first is on the master request and the next on a sub request. but that was very very flacky design from the beginning, as changing the template arrangement could have changed this and make everything look at the subrequest.

**Comment by dbu** [comment]
nice!

File: Block/ActionBlockService.php (hunk 1 above)
Comment (WouterJ) [inline Block/ActionBlockService.php]: Thanks, updated to use `getCurrentRequest()` now.

**Comment by dbu** [comment]
thanks a lot wouter!

**Pull request merged**